pagopa/pn-frontend
# chore(Pn-3895): check and update cookie scripts for all webapps

## Short description
Updated and checked every cookie script generated by OneTrust

## List of changes proposed in this pull request
- updated script for pn-pa-webapp
- updated script for pn-login-webapp
- updated script for pn-personafisica-webapp
- updated script for pn-personagiuridica-webapp

## How to test
Run each application and wait for cookie banner to pop-up (better to test with incognito mode).
Then go to "Scopri di più" and check cookie details to be configured correctly.
